api/api-doc/column_family.json

```diff
@@ -84,6 +84,14 @@
                      "type":"string",
                      "paramType":"path"
                   },
+                  {
+                     "name":"flush_memtables",
+                     "description":"Controls flushing of memtables before compaction (true by default). Set to \"false\" to skip automatic flushing of memtables before compaction, e.g. when the table is flushed explicitly invoking the compaction api.",
```

Comment:
I don't understand this explanation. What do you mean "e.g., when the table is flushed explicitly invoking the compaction api"? Did you forget the word **before**? I.e., "when the table was flushed explicitly **before** invoking the compacation api"?

In general, I don't like the phenomenon that recently we're adding too many obscure options to the REST API which solve in a manual way some very specific problems that nobody will remember in a year (or tomorrow) that even need to be solved. If these are important as manual options, they should be clearly documented in a document - not just one line. If they are not important as manual options, they should have been automatic.

Reply:
I see below that you have another copy of this JSON, and it has exactly the word "before" I was missing.
Please choose the best text for both.

Reply:
sure, will add "before".
The option is not needed if `scylla nodetool compact` is used, then everything is done automatically and efficiently.

Reply:
fixed in v7